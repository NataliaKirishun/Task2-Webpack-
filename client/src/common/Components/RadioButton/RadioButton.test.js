import React from 'react';
import { shallow } from 'enzyme';
import RadioButton from './RadioButton';

describe('RadioButton Snapshot', () => {
    it('RadioButton Component should render correctly', () => {
        const component = shallow(<RadioButton />);
        expect(component).toMatchSnapshot();
    });
});