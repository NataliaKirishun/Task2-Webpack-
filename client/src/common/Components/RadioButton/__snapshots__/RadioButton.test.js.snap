// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioButton Snapshot RadioButton Component should render correctly 1`] = `
<Fragment>
  <styled.input
    onChange={[Function]}
    type="radio"
  />
  <styled.label
    active={true}
  />
</Fragment>
`;
